packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/index.md

```diff
@@ -56,4 +62,6 @@ In addition to configuring your Okta org for the multifactor use case, you need
 
 ## Troubleshooting tips
 
-Ensure that the password recovery user is valid with an active user status in your Okta org.
\ No newline at end of file
+Verify that the password recovery user is valid with an active user status in your Okta org.
```

Comment:
This doesn't make sense wihtout context. What problems might the developer \ user have for which this is a tip to fix

Reply:
I currently tested "staged" and not "active" users and the password recovery worked. When this text was first written last year it may have had more sense but I agree there is little value to it now. I removed the text.